src/aiperf/orchestrator/strategies.py

```
@@ -0,0 +1,334 @@
+            disable_warmup_after_first: Disable warmup for runs after the first.
+                If True (default), only the first run includes warmup, subsequent
+                runs measure steady-state performance. If False, all runs include
+                warmup (useful for long cooldown periods or cold-start scenarios).
+                
+        Raises:
+            ValueError: If num_trials < 1 or cooldown_seconds < 0
+        """
+        if num_trials < 1:
+            raise ValueError(
+                f"Invalid num_trials: {num_trials}. Must be at least 1."
+            )
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)